src/Akka.Streams.Kafka.Tests/KafkaIntegrationTests.cs

```diff
@@ -1,19 +1,88 @@
 using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Akka.Actor;
 using Akka.Configuration;
+using Akka.Streams.Dsl;
+using Akka.Streams.Kafka.Dsl;
+using Akka.Streams.Kafka.Messages;
 using Akka.Streams.Kafka.Settings;
+using Akka.Streams.TestKit;
+using Confluent.Kafka;
 using Xunit;
 using Xunit.Abstractions;
+using Config = Akka.Configuration.Config;
 
 namespace Akka.Streams.Kafka.Tests
 {
     [Collection(KafkaSpecsFixture.Name)]
     public class KafkaIntegrationTests : Akka.TestKit.Xunit2.TestKit
     {
-        public KafkaIntegrationTests(string actorSystemName, ITestOutputHelper output) 
+        private readonly KafkaFixture _fixture;
+        protected IMaterializer Materializer { get; }
+
+        public KafkaIntegrationTests(string actorSystemName, ITestOutputHelper output, KafkaFixture fixture) 
```

Comment:
Makes sense to move all of this boilerplate code to the `KafkaIntegrationTests` fixture so it can be re-used.

Reply:
@Aaronontheweb Well, `KafkaIntegrationTests` is a base class for all kafka tests by itself, so all of this code is going to be reused. Maybe would be nice to make it `abstract` by the way... Is it what you were talking about?